app/services/audio_service.py

```
@@ -0,0 +1,377 @@
+            raise
+    
+    def _store_audio_cache(self, video_id: str, audio_path: str):
+        """Store audio cache info in database"""
+        try:
+            if self.db:
+                file_size = os.path.getsize(audio_path)
+                # Get audio duration
+                try:
+                    audio = AudioSegment.from_file(audio_path)  # Cambiado de from_wav a from_file para MP3
+                    duration = len(audio) / 1000.0  # Convert to seconds
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)